### PR TITLE
Feature/lever staging columns added

### PR DESCRIPTION
**Pull request by fivetran-avinash**
Pull Request
**Are you a current Fivetran customer?** 

**What change(s) does this PR introduce?** 

- Added column names in final staging model CTEs to follow best practices for model development.

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#14]
- [ ] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🌻 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/stg_lever__contact_email.sql [hunk 1]
```diff
@@ -19,8 +19,12 @@ fields as (
 ),
 
 final as (
-    
-    select *
+
+    select 
+        contact_id,
+        cast(_fivetran_synced as {{ dbt_utils.type_timestamp() }}) as _fivetran_synced,
+        email
```
Comment (fivetran-reneeli) [inline models/stg_lever__contact_email.sql:26]: While we're here making edits, we may as well update the models to fit the [style guide](https://github.com/fivetran/dbt_style_guide)! I'll edit one to start off with.

**Review by fivetran-reneeli — CHANGES_REQUESTED** [review]
Hey @fivetran-avinash great job! The only note I had was to update the models to follow our [style guide](https://github.com/fivetran/dbt_style_guide) while we're here. You can look at the `stg_lever__archive_reason` for example

File: models/stg_lever__archive_reason.sql [hunk 2]
```diff
@@ -31,4 +30,5 @@ final as (
     where not coalesce(_fivetran_deleted, false)
 )
 
-select * from final
+select *
```
Comment (fivetran-reneeli) [inline models/stg_lever__archive_reason.sql:33]: I applied some changes to follow our style guide here to use an example!

Comment (fivetran-avinash) [reply models/stg_lever__archive_reason.sql:33]: Thanks @fivetran-reneeli!

**Comment by fivetran-avinash** [comment]
@fivetran-reneeli Requested changes applied!

**Review by fivetran-reneeli — APPROVED** [review]
Looks great!

**Pull request merged**